main/src/ca/uwaterloo/flix/language/ast/NamedAst.scala

```diff
@@ -46,9 +46,9 @@ object NamedAst {
 
     case class Def(sym: Symbol.DefnSym, spec: Spec, exp: Expr) extends Declaration
 
-    case class Enum(doc: Ast.Doc, ann: Ast.Annotations, mod: Ast.Modifiers, sym: Symbol.EnumSym, tparams: TypeParams, derives: List[Name.QName], cases: List[Declaration.Case], loc: SourceLocation) extends Declaration
+    case class Enum(doc: Ast.Doc, ann: Ast.Annotations, mod: Ast.Modifiers, sym: Symbol.EnumSym, tparams: TypeParams, derives: List[Name.QName], derivesLoc: SourceLocation, cases: List[Declaration.Case], loc: SourceLocation) extends Declaration
```

Comment:
Here I would still want a `Derivations` class, but instead I would put it inside `NamedAst.scala` near the bottom of the file.

E.g.

```
case class Derivations(classes: List[Name.QName], loc: SourceLocation)
````

The same applies for ParsedAst, WeededAst, etc.

Reply:
Makes sense?

Reply:
Yeah 👍